dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ConjureBodySerDe.java

```diff
@@ -258,6 +258,12 @@ public T deserialize(Response response) {
                 // deserializer has taken on responsibility for closing the response body
                 closeResponse = false;
                 return deserialized;
+            } catch (IOException e) {
+                throw new SafeRuntimeException(
```

Comment:
For posterity: this does not get retried, it will just fail to the client. It's just that for consistency we're wrapping the exception here instead of Deserializer impls.

Reply:
Fixing this would require a lot more work.